go.mod

```diff
@@ -49,16 +49,17 @@ require (
 	gopkg.in/inf.v0 v0.9.1
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 	gopkg.in/yaml.v2 v2.2.2
-	istio.io/api v0.0.0-20190820204432-483f2547d882
-	k8s.io/api v0.0.0-20190620084959-7cf5895f2711
-	k8s.io/apimachinery v0.0.0-20190612205821-1799e75a0719
+	istio.io/api v0.0.0-20191115173247-e1a1952e5b81
+	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
```

Comment:
I know this is getting replaced but this pseudo version looks off

Reply:
I agree, it only looks slightly newer than the previous one. I'm not sure why it's grabbing this one, but like you mentioned it is getting replaced.